pages/api/books.ts

```diff
@@ -130,3 +131,27 @@ export const removeToFinishedBooks = (bookId: number) => {
     return res.data
   })
 }
+

```

Comment:
better to just remove this if this is not being used. Make sure to avoid leaving commented code on PRs. So, if it's getting used, it has to not be commented out.